src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -56,8 +58,16 @@ public function __invoke(SendRequest $sendRequest): void
             $this->fail($sendRequest->id, "Validation failed: {$this->validator->getReason()}");
         }
 
-        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
-        $bodyPlainText = strip_tags($bodyRenderedHtml);
+        $email = new Swift_Message();
+
+        $additionalParams['headerImageRef'] = $this->embedImages($email, 'TBG.png');
+        $additionalParams['footerImageRef'] = $this->embedImages($email, 'CCh.png');
+        $additionalParams['renderHtml'] = true;
+
+        $templateMergeParams = array_merge($additionalParams, $sendRequest->params);
+
+        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $templateMergeParams);
+        $bodyPlainText = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
```

Comment:
@tbg-jasonfung  We still need to also `strip_tags` since the inner template is HTML.

I've added this and the spacing is weird but otherwise it works, so I think it's good enough for emails to have an accessible fallback that doesn't look too broken.